composer.json

```diff
@@ -35,20 +35,19 @@
         "guzzlehttp/guzzle": "^7.2",
         "mockery/mockery": "^1.3.2",
```

Comment:
the lowest build fails with a fatal error on prophecy. is there a newer version of mockery that we can require as minimal version to avoid the issue? (i think this is where prophecy is coming from)